corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -116,6 +116,16 @@
 
 {% include 'app_manager/v1/partials/case_list_filtering.html' %}
 
+{{ request|toggle_tag_info:"DETAIL_LIST_TAB_NODESETS" }}
+{% if request|toggle_enabled:'DETAIL_LIST_TAB_NODESETS' %}
+<div data-bind="with: shortScreen">
```

Comment:
i moved this under Case Details. The changes in v1 and v2 are slightly different. Had to add a spacer div under v2 and may be we have discontinued using that for spacing now.